crates/uv-keyring/src/windows.rs

```diff
@@ -163,13 +163,19 @@ impl CredentialApi for WinCredential {
         let target_name = to_wstr(&self.target_name);
         let cred_type = CRED_TYPE_GENERIC;
         // SAFETY: Calling Windows API
-        match crate::blocking::spawn_blocking(move || unsafe {
-            Ok(CredDeleteW(target_name.as_ptr(), cred_type, 0))
+        let winerr: u32 = crate::blocking::spawn_blocking(move || unsafe {
+            Ok(if CredDeleteW(target_name.as_ptr(), cred_type, 0) != 0 {
+                0 // success
+            } else {
+                GetLastError()
+            })
         })
-        .await?
-        {
-            0 => Err(decode_error()),
```

Comment:
Notice here we called `decode_error` which invokes `GetLastError` _outside_ of the credential delete thread

Reply:
cc @oconnor663 just so you're aware of the gotcha in the future :)